### DISCUSSION TITLE
Unit Redirection Issue

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#713] [General Discussion]
Hello Team,

We are on release 5.0
We are encountering an issue with unit redirection within the course platform. Some users have reported that upon clicking on a particular unit, they are being redirected to a different unit instead.
We also encountered that this issue is with the collaborator  which we added to the course. Any content that is added by the collaborators is facing the issue of redirection. 

To provide further context and assist in resolving this matter promptly, I have attached a video demonstrating the problem. Kindly review the video for reference.

https://github.com/Sunbird-Ed/Community/assets/72926048/c0f4080d-9df2-4a7b-9193-fffc5ac24735

Thank you for your attention to this matter. If you require any additional information or assistance, please don't hesitate to reach out.

**Comment by anisha147** [comment]
Hi @siddhishinde0723 

The team is looking into the matter internally and will provide an update once it is resolved. 
Please refer to this [thread](https://github.com/orgs/Sunbird-Ed/discussions/721) for the same.

Thanks!

cc: @swayangjit @rajeevsathish @vinukumar-vs @ashokreddy1208 @pallakartheekreddy
